programs/pre-authorized-debit-v1/src/lib.rs

```diff
@@ -6,6 +6,17 @@ pub mod state;
 
 use instructions::*;
 
+#[cfg(not(feature = "no-entrypoint"))]
+solana_security_txt::security_txt! {
+    name: "Pre Authorized Debit Program",
+    project_url: "https://docs.seabed.so/open-source-primitives/pre-authorized-debit-v1",
+    contacts: "email:mocha@dcaf.so,email:matcha@dcaf.so,link:https://github.com/dcaf-labs/pre-authorized-debit/security/advisories/new",
+    policy: "https://github.com/dcaf-labs/pre-authorized-debit/blob/main/SECURITY.md",
+    preferred_languages: "en",
+    source_code: "https://github.com/dcaf-labs/pre-authorized-debit",
+    auditors: "None"
+}
+
```

Comment:
Make the following changes:
1. Add a commit or tag like this https://github.com/Squads-Protocol/squads-mpl/blob/main/programs/squads-mpl/src/lib.rs#L35C50-L35C90
2. Make the following suggested changes

```suggestion
#[cfg(not(feature = "no-entrypoint"))]
solana_security_txt::security_txt! {
    name: "Pre Authorized Debit Program",
    project_url: "https://docs.seabed.so/open-source-primitives/pre-authorized-debit-v1",
    contacts: "email:mocha@dcaf.so,email:matcha@dcaf.so,link:https://github.com/seabed-labs/pre-authorized-debit/security/advisories/new",
    policy: "https://github.com/seabed-labs/pre-authorized-debit/blob/main/SECURITY.md",
    preferred_languages: "en",
    source_code: "https://github.com/seabed-labs/pre-authorized-debit",
    auditors: "None"
}

```

Reply:
Done